build_package.py

```diff
@@ -930,6 +962,15 @@ def build_base_files(base_configs="config"):
 
     make_soft_link("../py/scalyr_agent/agent_main.py", "scalyr-agent-2")
     make_soft_link("../py/scalyr_agent/config_main.py", "scalyr-agent-2-config")
+    # add switch python version script.
+    shutil.copy(
+        os.path.join(agent_source_root, "installer", "scripts", "scalyr-switch-python.sh"),
+        "scalyr-switch-python"
```

Comment:
I wonder if for consistency with other binaries we should call this one ``scalyr-agent-2-switch-python``.

Reply:
I think I agree with you. 
@czerwingithub what do you think about this name?